meinberlin/templates/a4reports/emails/report_moderators.en.email

```diff
@@ -1,2 +1,31 @@
 {# Hack to ensure german mails are sent regardless of the current users browser language setting #}
 {% extends 'a4reports/emails/report_moderators.de.email' %}
+
+{% comment %}
```

Comment:
You added it as a comment, so that we have the text, because it was translated?

Reply:
Well it was only the 2 upper lines before, which I removed and filled the email and then realized that it is actually sent instead of the german one (which is the reason for the hack obviously, haha). And then I just kept what I did as a comment, because it was already there.. should I delete it?

Reply:
No, leave it. Good idea!